### ISSUE TITLE
[cranelift] CONTRIBUTING.md is a dead link

### ISSUE THREAD

**Issue by jyn514** [bytecodealliance/wasmtime#1280]
Probably got lost in the switch to wasmtime. [`cranelift/README.md`](https://github.com/bytecodealliance/wasmtime/blob/master/cranelift/README.md#contributing) links to [`cranelift/CONTRIBUTING.md`](https://github.com/bytecodealliance/wasmtime/blob/master/cranelift/CONTRIBUTING.md), but there's no file there. Not sure whether you want this to link to [`CONTRIBUTING.md`](https://github.com/bytecodealliance/wasmtime/blob/master/CONTRIBUTING.md) or something in [`cranelift/docs`](https://github.com/bytecodealliance/wasmtime/tree/master/cranelift/docs), so I didn't make a PR.

**Comment by abrown** [comment]
Linking to the wasmtime-level CONTRIBUTING.md would make sense to me.

**Comment by jyn514** [comment]
Is there a reason not to add back the old CONTRIBUTING.md file? https://github.com/bytecodealliance/cranelift/blob/3d1cdec0837684b1656bcb6194be6c31302b5d3e/CONTRIBUTING.md

**Comment by abrown** [comment]
I suspect it was removed to remove duplication; @alexcrichton?

**Comment by abrown** [comment]
@jyn514, what does the cranelift file have that is missing in the wasmtime file? (I mean what information do you feel is important and missing?)

**Comment by jyn514** [comment]
Actually, looking back at both it seems like they cover most of the same info, so I'd be ok with only keeping the wasmtime website. I was a little put off by most of the sections on the website being missing, but 10. Contributing all seems to be there.

**Comment by alexcrichton** [comment]
Oh oops sorry about the broken link! I removed the one in cranelift and [merged it with the wasmtime contributing docs](https://bytecodealliance.github.io/wasmtime/contributing.html). The top-level CONTRIBUTING.md is just a thin shim pointing to those docs. I figured there wasn't much use in having two CONTRIBUTING.md files, but I think it's fine for the README to point directly to the online docs!
